src/main/java/org/apache/datasketches/kll/KllDoublesHelper.java

```diff
@@ -0,0 +1,366 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.datasketches.kll;
+
+import static org.apache.datasketches.Util.floorPowerOf2;
+
+import java.util.Arrays;
+import java.util.Random;
+
+import org.apache.datasketches.SketchesArgumentException;
+
+/**
+ * Static methods to support KllSketch
+ * @author Kevin Lang
+ * @author Alexander Saydakov
+ */
+public class KllDoublesHelper {
+
+  static boolean isEven(final int value) {
```

Comment:
perhaps, such methods, identical in both KllFloatsHelper and KllDoublesHepler, should be shared in common helper, say, KllHelper?